test/e2e/install_test.go

```diff
@@ -272,90 +333,95 @@ var _ = Describe("Operator Install", func() {
 // - bundle
 // - bundledeployments
 // - catalogsources
-func getArtifactsOutput(ctx context.Context, basePath string) {
+func getArtifactsOutput(t *testing.T) {
+	basePath := env.GetString("ARTIFACT_PATH", "")
+	if basePath == "" {
+		return
+	}
+
 	kubeClient, err := kubeclient.NewForConfig(cfg)
-	Expect(err).To(Not(HaveOccurred()))
+	require.NoError(t, err)
 
 	// sanitize the artifact name for use as a directory name
-	testName := strings.ReplaceAll(strings.ToLower(CurrentSpecReport().LeafNodeText), " ", "-")
+	testName := strings.ReplaceAll(strings.ToLower(t.Name()), " ", "-")
 	// Get the test description and sanitize it for use as a directory name
 	artifactPath := filepath.Join(basePath, artifactName, fmt.Sprint(time.Now().UnixNano()), testName)
 
 	// Create the full artifact path
 	err = os.MkdirAll(artifactPath, 0755)
-	Expect(err).To(Not(HaveOccurred()))
+	require.NoError(t, err)
 
 	// Get all namespaces
 	namespaces := corev1.NamespaceList{}
-	if err := c.List(ctx, &namespaces); err != nil {
-		GinkgoWriter.Printf("Failed to list namespaces %w", err)
+	if err := c.List(context.Background(), &namespaces); err != nil {
```

Comment:
Would it be worth creating a single context to reuse in all these calls instead of using `context.Background()` in every call?

Reply:
Not sure it matters much since there will never be a context in test code